husky_components/column-navigation/main.js

```diff
@@ -446,8 +459,15 @@ define(function() {
 
             this.columns = [];
             this.selected = [];
-            // array with all marked ids
-            this.marked = this.options.premarkedIds || [];
+
+            this.marked = {};
+            // write premerkedIds in this.marked
```

Comment:
`premarkedIds`, or should we delete this comment completely? Doesn't feel like it adds a huge value.